lvm.go

```diff
@@ -98,6 +101,14 @@ type LV struct {
 
 	// Encrypted indicates if the logical volume is encrypted.
 	Encrypted bool `json:"encrypted"`
+
+	// DecryptedLVName is the name of the decrypted logical volume as set by
+	// the CryptOpen call.
+	DecryptedLVName string `json:"decryptedLVName"`
```

Comment:
DecryptedPath is really all you need. there isn't a specific volume for the decrypted path.

Reply:
the cryptopen command actually took the name "lv0_enc" and the path is "/dev/mapper/lv0_enc" thought capturing both is a good idea.

Reply:
What i also noticed is if we "-" in the name the path actually becomes "--".

Reply:
I'd really like to make this cleaner, so users of the api didn't have to really think about "if it is an encrypted volume than do this... otherwise"...

it just seems like it can be simpler than it is.

i wont block this as it is though.

Reply:
@smoser ok let me merge this an you can propose the cleaned up version.